src/Resolvers/Microsoft.DotNet.MSBuildSdkResolver/Microsoft.DotNet.MSBuildSdkResolver.csproj

```diff
@@ -36,6 +36,7 @@
     <ItemGroup>
       <ManifestLines Include="&lt;SdkResolver&gt;" />
       <ManifestLines Include="&lt;Path&gt;$(_TargetPathRelativePath)&lt;/Path&gt;" />
+      <ManifestLines Include="&lt;ResolvableSdkPattern&gt;Microsoft\..*|Samsung\..*|GtkSharp\..*|FSharp\.NET\.Sdk|NuGet\.Build\.Tasks\.Pack&lt;/ResolvableSdkPattern&gt;" />
```

Comment:
better escape the `*` too or I think we'll try to glob it.

Reply:
I don't see it globbed in the generated output?

```xml
<SdkResolver>
<Path>..\..\Microsoft.DotNet.MSBuildSdkResolver.dll</Path>
<ResolvableSdkPattern>Microsoft\..*|Samsung\..*|GtkSharp\..*|FSharp\.NET\.Sdk|NuGet\.Build\.Tasks\.Pack</ResolvableSdkPattern>
</SdkResolver>
```
?

Reply:
I was expecting it to try to glob, fail, and fall back to the literal text. But procmon doesn't see an enumeration attempt either. I'm not sure why--maybe we see the angle brackets and decide it's not a path before globbing?